geoengine/types.py

```diff
@@ -157,6 +157,30 @@ def from_response(response: Dict[str, Any]) -> None:
         raise TypeException(
             f'Unknown `ResultDescriptor` type: {result_descriptor_type}')
 
+    @classmethod
+    def is_raster_result(cls) -> bool:
+        '''
+        Return true if the result is of type raster
+        '''
+
+        return False
```

Comment:
rather: `raise NotImplementedError()`
this should be a abstract method imho

Reply:
No, this way you only have to implement your specific method that returns True.